VideoLocker/res/layout/view_progress_download_btn.xml

```diff
@@ -15,16 +14,9 @@
         android:layout_width="25dp"
         android:layout_height="25dp"
```

Comment:
p.p.s. if its out of scope of this PR, we can do it on any other day when the sun shines brightly :wink: